docs/CHANGELOG.asciidoc

```diff
@@ -74,6 +75,49 @@ Review important information about the {kib} 7.17.x releases.
 
 --
 
+[[release-notes-7.17.4]]
+== {kib} 7.17.4
+
+coming::[7.17.4]
+
+Review the following information about the 7.17.4 release.
+
+[float]
+[[breaking-changes-v7.17.4]]
+=== Breaking changes
+
+// tag::notable-breaking-changes[]
+There are no breaking changes in the 7.17.4 release.
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following: 
+
+{kibana-ref-all}/7.16/release-notes-7.16.0.html#breaking-changes-7.16.0[7.16] | {kibana-ref-all}/7.15/release-notes-7.15.0.html#breaking-changes-7.15.0[7.15] | {kibana-ref-all}/7.14/release-notes-7.14.0.html#breaking-changes-7.14.0[7.14] | {kibana-ref-all}/7.13/release-notes-7.13.0.html#breaking-changes-7.13.0[7.13] | 
+{kibana-ref-all}/7.12/release-notes-7.12.0.html#breaking-changes-7.12.0[7.12] | {kibana-ref-all}/7.11/breaking-changes-7.11.html[7.11] | 
+{kibana-ref-all}/7.10/breaking-changes-7.10.html[7.10] | {kibana-ref-all}/7.9/breaking-changes-7.9.html[7.9] | {kibana-ref-all}/7.8/breaking-changes-7.8.html[7.8] | 
+{kibana-ref-all}/7.7/breaking-changes-7.7.html[7.7] | {kibana-ref-all}/7.6/breaking-changes-7.6.html[7.6] | {kibana-ref-all}/7.5/breaking-changes-7.5.html[7.5] |
+{kibana-ref-all}/7.4/breaking-changes-7.4.html[7.4] | {kibana-ref-all}/7.3/breaking-changes-7.3.html[7.3] | {kibana-ref-all}/7.2/breaking-changes-7.2.html[7.2] | 
+{kibana-ref-all}/7.1/breaking-changes-7.1.html[7.1] | {kibana-ref-all}/7.0/breaking-changes-7.0.html[7.0]
+
+[float]
+[[enhancement-v7.17.4]]
+=== Enhancements
+Alerting::
+Adds the `xpack.actions.email.domain_allowlist` configuration setting for the `actions` plugin {kibana-pull}129001[#129001]
+
+Monitoring::
+* Adds the ability to collect Telemetry {kibana-pull}130498[#130498]
+* Adds the ability to report dashboard panels by type {kibana-pull}130166[#130166]
+
+[float]
+[[fixes-v7.17.4]]
+=== Bug Fixes
+Alerting::
+Revert back the support of preconfigured connectors for Cases that was mistakenly removed in 7.17.0 {kibana-pull}130372[#130372]
```

Comment:
I think it is better to match the context here https://github.com/elastic/security-docs/pull/1898/files#diff-989eebfd38dc3e23394a16f72f15c414be7eb0bca343d76bffa117a93e65c3daR12. I think it is better to avoid why it was removed in the first place. What do you think about it @peteharverson @sophiec20 ?

Reply:
Yes good point. Should just say `Revert back the support of preconfigured connectors for Cases.`